python/lsst/meas/base/baseMeasurement.py

```diff
@@ -433,6 +422,46 @@ def doMeasurement(self, plugin, measRecord, *args, **kwds):
                 plugin.name, measRecord.getId(), error)
             plugin.fail(measRecord)
 
+
+class BaseMeasurementTask(SimpleBaseMeasurementTask):
+    """Ultimate base class for all measurement tasks.
```

Comment:
This docstring is no longer true.  Could you also specify above what is `Simple` there and not simple here?